app/assets/javascripts/protocols/steps.js.erb

```diff
@@ -79,7 +79,6 @@ function applyCancelCallBack() {
 
     setTimeout(function() {
       initStepsComments();
-      openLinksInNewTab();
```

Comment:
This code was added because quill didn't add the target blank attribute to the link tag and since we don't have quill anymore I presume that we can throw out this